src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala

```diff
@@ -24,17 +27,44 @@ object ToolFormApp extends App {
     for {
       toolConfiguration <- parseCommandLineArgs(args)
       project           <- readProject(toolConfiguration.in)
-      summary           <- projectSummary(project)
-    } yield summary
+      result <- toolConfiguration.mode match {
+        case ToolFormAppMode.Inspect  => new InspectTask().run(toolConfiguration, project)
+        case ToolFormAppMode.Generate => new GenerateTask().run(toolConfiguration, project)
+      }
+    } yield result
 
   private def parseCommandLineArgs(args: Array[String]): Either[ToolFormError, ToolFormConfiguration] = {
     val parser = new scopt.OptionParser[ToolFormConfiguration]("toolform") {
-      head("toolform", "0.1")
+      head(PackageInfo.getPackageName, PackageInfo.getPackageVersion)
```

Comment:
I had the same thought here and used sbt-buildinfo to generate this.  Sorry about the conflict.  Should easy to fix up...

Reply:
sbt-buildinfo is probably a better way to do it because it is compile time. This reflection method doesn't seem to work in test environments either. I just didn't know if I should be pulling in dependencies or not.